### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by ghost**
- Ran through chat gpt to improve readability
- Added `README.md` in path `example-projects`
- Generated Korean docs for example projects
- Added Example projects into `SUMMARY.md`
- Update markdown file names using camelCase to kebab-case for consistency

### CODE REVIEW

File: kr/developers-guide/example-projects/mystery-box-example.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+이 프로젝트의 코드는 [mystery box orakl](https://github.com/Bisonai/orakl-demo-mystery-box)에서 확인할 수 있습니다. 자세한 지침은 readme.md 파일을 참조하십시오.
+
+## NFT 상자 계약
+
+이 계약은 NFT에 대한 ERC-721 표준을 준수하며 추가 기능이 포함되어 있습니다.
+
+### 상자 열기
+
+openBox 함수는 Orakl VRF 서비스에서 무작위 숫자를 요청하고 NFT를 태우며 사용자에게 해당하는 NFT를 보상합니다.
```
Comment (bryan-bisonai) [inline kr/developers-guide/example-projects/mystery-box-example.md:17]: To unify the convention,

openBox -> openBox()

File: kr/developers-guide/example-projects/mystery-box-example.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    _requireOwned(_tokenId);
+    uint256 requestId = requestRandomWords();
+    requestIdToPlayer[requestId] = msg.sender;
+    _burn(_tokenId);
+}
+```
+
+### 무작위 숫자 처리
+
+fulfillRandomWords 함수는 Orakl VRF 서비스에서 무작위 숫자를 받아 해당하는 NFT를 생성합니다.
```
Comment (bryan-bisonai) [inline kr/developers-guide/example-projects/mystery-box-example.md:30]: fulfillRandomWords -> fulfillRandomWords()

**Review by bryan-bisonai — COMMENTED** [review]
LGTM! ;)

**Review by martinkersner — APPROVED** [review]
LGTM!

**Pull request merged**